delta/sdk/src/test/scala/ch/epfl/bluebrain/nexus/delta/sdk/resources/ResourcesSpec.scala

```diff
@@ -331,7 +331,8 @@ class ResourcesSpec
             source = newSource,
             updatedAt = time2,
             updatedBy = subject,
-            types = newTypes
+            types = newTypes,
+            schema = schemaRev
```

Comment:
I believe this is because the `ResourceState` allows the schema to be a `ResourceRef` while in a `ResourceEvent` it has to specifically be a `ResourceRef.Revision`. This shouldn't be a big change if we want to change it 🤔

Reply:
Note that we will likely change part of how we model this to fix: https://github.com/BlueBrain/nexus/issues/3401